Detections/CommonSecurityLog/MultiVendor-PossibleDGAContacts.yaml

```diff
@@ -111,4 +111,4 @@ query: |
     )
     on Name
     | project StartTime, EndTime, Name, DGADomain, SourceIP, DestinationIP, DataSource
-    | extend timestamp=StartTime, IPCustomEntity=SourceIP
\ No newline at end of file
+    | extend timestamp=StartTime, IPCustomEntity=SourceIP, HostCustomEntity=Name
```

Comment:
Name in the instance is a Domain Name, not a Computer or Host Name.  We should not be pushing thru the Domain into this entity field.  We will soon have the ability to have a DNS DomainName field.  You can see the new changes coming here - https://github.com/Azure/Azure-Sentinel/pull/1319